src/renderer/contexts/ExecutionContext.tsx

```diff
@@ -201,6 +213,17 @@ export const ExecutionProvider = memo(({ children }: React.PropsWithChildren<{}>
         [setStatus, unAnimate, schemata]
     );
 
+    useBackendEventSourceListener(
+        eventSource,
+        'node-output-data',
+        (data) => {
+            if (data) {
+                setOutputDataMap((prev) => new Map({ ...prev, [data.nodeId]: data.data }));
```

Comment:
We want to iterate the values of the map, not its object properties.

```suggestion
                setOutputDataMap((prev) => new Map([ ...prev, [data.nodeId]: data.data ]));
```

Reply:
This doesn't work.

Reply:
My bad, New suggestion.